Sources/Plasma/NucleusLib/pnModifier/plLogicModBase.h

```diff
@@ -76,13 +76,13 @@ class plLogicModBase : public plSingleModifier
     bool                    fDisabled;
 
     bool IEval(double secs, float del, uint32_t dirty) override { return false; }
-    void IUpdateSharedState(bool triggered) const;
     void IHandleArbitration(class plServerReplyMsg* msg);
     bool IEvalCounter();
     virtual void PreTrigger(bool netRequest);
     virtual void Trigger(bool netRequest);
     virtual void UnTrigger();
-    
+    virtual void UpdateSharedState(bool triggered) const { }
```

Comment:
This may be a bit of a personal opinion, but I think this should probably be a pure virtual function. Generally, logic modifiers are waiting for the server response here, so, having a blank implementation in some theoretical non-`plLogicModifier` subclass would leave those `plLogicModBase` instances dead in the water.

Reply:
Updated to be pure virtual